src/components/Filters/FilterChecklist.vue

```diff
@@ -188,6 +199,12 @@ export default {
   margin-bottom: 0.5rem;
 }
 
+.filter-heading {
+  font-size: 1rem;
+  font-weight: 600;
+  margin-bottom: 16px;
```

Comment:
```suggestion
  margin-bottom: 1rem;
```

https://nekocalc.com/px-to-rem-converter